tests/sharding/test_vmc_handler.py

```diff
@@ -304,13 +307,56 @@ def get_testing_colhdr(vmc_handler,
     ])
 
 
+@pytest.mark.parametrize(  # noqa: F811
+    'mock_score,mock_is_new_head,expected_score,expected_is_new_head',
+    (
+        # test case in doc.md
+        (
+            (10, 11, 12, 11, 13, 14, 15, 11, 12, 13, 14, 12, 13, 14, 15, 16, 17, 18, 19, 16),
+            (True, True, True, False, True, True, True, False, False, False, False, False, False, False, False, True, True, True, True, False),  # noqa: E501
+            (19, 18, 17, 16, 16, 15, 15, 14, 14, 14, 13, 13, 13, 12, 12, 12, 11, 11, 11, 10),
+            (True, True, True, True, False, True, False, True, False, False, True, False, False, True, False, False, True, False, False, True),  # noqa: E501
+        ),
+        (
+            (1, 2, 3, 2, 2, 2),
+            (True, True, True, False, False, False),
+            (3, 2, 2, 2, 2, 1),
+            (True, True, False, False, False, True),
+        ),
+    )
+)
+def test_vmc_fetch_candidate_head(vmc,
```

Comment:
It's better to be aligned with https://github.com/pipermerriam/ethereum-dev-tactical-manual/blob/master/style-guide.md#function-signatures